makemkv.js

```diff
@@ -224,7 +224,7 @@ class MakeMkv {
         };
 
         this.ripTrack(
-            saveDirectory, driveId, this.ripQueue.driveId.pop(), newCallback
+            saveDirectory, driveId, Array.from(this.ripQueue.driveId).pop(), newCallback
```

Comment:
Hoping this actually removes the last element from the set and returns it; rather than just returning it. Hoping to test drive some of this stuff later on, but just trying to get things working quick-and-dirty for now with minimal change.

Reply:
It doesn't, and you can't access sets by index either. Better would probably be to just switch the set back to an array - it was just to clear up an edge case when someone submits the same track twice. It would only be possible if there are two UIs open talking to the server at the same time, and both submit the same track for ripping. Totally not going to happen outside of a lab

Reply:
But if you did want to handle this:

```
const ripTracks = [...this.ripQueue.driveId];
const ripTrack = ripTracks.pop();
this.ripQueue.driveId.delete(ripTrack);
```

Seems like a lot of work though, when we could accomplish the same functionality by just guarding the array push for the track